libraries/APM_Control/AP_AutoTune.h

```diff
@@ -1,6 +1,5 @@
 #pragma once
 
-#include <AP_Logger/AP_Logger.h>
```

Comment:
can remove logstructure.h here too, if you remove log_ATRP

Reply:
Yeah, can't do that as that message is included in ArduPlane's LogStructure!

Should probably be a WriteStreaming call instead, but for now I've put it back in the header, where we only need `AP_Logger/LogStructure.h`, not `AP_Logger.h`, which is the important change.